tests/ecosystem/deployment/test_acm.py

```diff
@@ -11,3 +11,10 @@
 @acm_import
 def test_acm_import():
     import_clusters_with_acm()
+
+
+@purple_squad
+@acm_import
```

Comment:
Suppose to be acm_install and not import.

Reply:
done